judges/assignee-was-attached/assignee-was-attached.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 Zerocracy
+# SPDX-License-Identifier: MIT
+
+# Judge that monitors open issue and create missing issue-was-assigned facts
+
+require 'fbe/conclude'
+require 'fbe/octo'
+
+Fbe.conclude do
+  quota_aware
+  on "(and (eq where 'github') (eq what 'issue-was-opened'))"
+
+  consider do |f|
+    next if Fbe.fb.query(
```

Comment:
@Yegorov we don't need to check this. Even if the issue is already closed, we are interested to know who it was assigned to. In order to reward.

Reply:
@yegor256 We need to fix it so that if a issue is already assigned, we don't check in GitHub API anymore?

Reply:
@Yegorov the marker is the presence of `issue-was-assigned`. If it's present, we should not check again. Later, when we register `issue-was-closed` fact, we should do an extra check -- who it is assigned to -- and replace the `issue-was-assigned` fact with a new one (if assignee is changed).

Reply:
@yegor256 thanks, fixed.